packages/react-server-integration-tests/src/__tests__/css/CssSpec.js

```diff
@@ -0,0 +1,35 @@
+var helper = require("../../specRuntime/testHelper");
+
+describe("A CSS page", () => {
+
+	helper.startServerBeforeAll(__filename, [
+		"./CssPage",
+	]);
+
+	helper.stopServerAfterAll();
+
+	describe("can apply a css class", () => {
+		helper.testWithDocument("/css", (document) => {
+			expect(document.querySelector("head link[rel=stylesheet]").href).toMatch(/route0\.css/);
+		});
+	});
+
+});
+
+describe("A CSS page with assets", () => {
+
+	helper.startServerBeforeAll(__filename, [
+		"./CssWithAssetsPage",
+	]);
+
+	helper.stopServerAfterAll();
+
+	describe("can apply a css class", () => {
+		helper.testWithDocument("/cssWithAssets", (document) => {
+      // this test is really just to make sure that react-server-cli doesn't explode
+      // when compiling CSS that includes images and fonts.
+			expect(document.querySelector("head link[rel=stylesheet]").href).toMatch(/route0\.css/);
```

Comment:
Sigh. There's a way but not easy. Also, I'm lazy.

I first wrote this test to check the computed styles in the HTML to see if the CSS was applied, but it turns out that ZombieJS doesn't download CSS 😢. So, I think the good way to do this would be to download the CSS with node's `http` module and insert the text into a `<style>` node, then test the computed style. Then, for styles that are of the form `url(foo.bar)`, try to download the asset referenced.

However, the test as written was failing and showing the bug that @withinboredom found, and I was too lazy to do all of that in this PR. Can we leave that for down the road? 😬